### ISSUE TITLE
Execution Block ID uid://A001/X15a0/X110 Sgr_A_st_s_03_TP

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#262]
Sgr_A_st_s_03_TP
[uid://A001/X15a0/X110](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/X110)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_s_03_TP/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_s_03_TP/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines

  * [ ] SPW16 H13CN
  * [ ] SPW18 H13CO+/SiO
  * [ ] SPW20 HCO+
  * [ ] SPW22 HNCO
  * [ ] SPW24 Cont 1
  * [ ] SPW26 Cont 2

**Comment by pyhsiehATalma** [comment]
Ripples in spw 27 (almost all EBs)

![spectral_plot_after_subtraction_uid___A002_Xfed4ee_Xee4 ms atmcor_Sgr_A_star_ant3_spw27_pol1](https://user-images.githubusercontent.com/95929782/199736423-b27ecdd5-4c8d-4aa9-8341-b4ed5ba484a4.png)

**Comment by pyhsiehATalma** [comment]
negative emission in spw 21 (HCO+). I have a feeling that it is due to baseline subtraction as we can see the baseline of HCO+ around 89.155 GHz is also negative, but not much we could do in this case unless we redo baseline subtraction. It doesn't help much though.

![uid___A001_X15a0_X110 s13_50 Sgr_A_star_sci spw21 cube I PM04 iter0 image sd pol0_ChannelMap_6](https://user-images.githubusercontent.com/95929782/199736534-6e04bba5-f694-42e5-b0b4-a651d56a89bc.png)

**Comment by pyhsiehATalma** [comment]
@keflavich  excepts two issues reported above, I didn't find anything wrong. I moved this SB to "Completed execution blocks (imaging, QA, calibration all done)". Please let me know if the workflow is ok.

**Comment by keflavich** [comment]
Yes, this seems fine.  At least, these are done until we come up with a better way to handle baseline subtraction

**Comment by eacmills** [comment]
Remaining baseline fitting issues for HCO+

Before/After
![spectral_plot_before_subtraction_uid___A002_Xfeb5e6_X6d6 ms atmcor_Sgr_A_star_ant0_spw21_pol0](https://user-images.githubusercontent.com/67716646/218235867-a8c0a940-3072-4342-a1f8-b0f94e3b3393.png)

![spectral_plot_after_subtraction_uid___A002_Xfeb5e6_X6d6 ms atmcor_Sgr_A_star_ant0_spw21_pol0](https://user-images.githubusercontent.com/67716646/218235871-7045fb63-0de6-4aa4-8abd-8a8796a2bab6.png)
